Portfolio/index.css

```diff
@@ -1 +1,88 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+  
+    margin:  0px;

```

Comment:
The value for padding is missing a unit (e.g., px). It should be 'padding: 10px;' for consistency with other declarations.